master/types.html

```diff
@@ -1271,7 +1271,7 @@ <h3 id="ListInterfaces">List interfaces</h3>
   </li>
 </ol>
 
-<p>Whenever a list element object is to be <dfn id="TermDetach">detached</dfn>,
+<p>Whenever a list element object is to be <dfn id="TermDetach" data-dfn-type="dfn" data-dfn-export>detached</dfn>,
```

Comment:
There are some like these that are very generic words. Not sure if they should be exported or have more descriptive labels applied.